release/release_data_tests.yaml

```diff
@@ -312,19 +312,39 @@
 # Map tests
 ###########
 
-- name: map_batches_benchmark_single_node
+- name: map
   run:
-    # Expect the benchmark to finish around 12 minutes.
-    timeout: 1080
-    script: python iter_batches_benchmark.py
+    timeout: 1800
+    script: python map_benchmark.py --api map --sf 10
+
+- name: flat_map
+  run:
+    timeout: 1800
+    script: python map_benchmark.py --api flat_map --sf 10
+
+- name: map_batches
+  run:
+    timeout: 5400
 
   variations:
-    - __suffix__: aws
-    - __suffix__: gce
-      env: gce
-      frequency: manual
-      cluster:
-        cluster_compute: single_node_benchmark_compute_gce.yaml
+    - __suffix__: numpy
+      # FIXME: Investigate why this is slow and OOMs.
+      stable: False
+      run:
+        script: >
+          python map_benchmark.py --api map_batches --batch-format numpy --sf 1000
+    - __suffix__: pandas
+      run:
+        script: >
+            python map_benchmark.py --api map_batches --batch-format pandas --sf 1000
```

Comment:
Extreme Nit: This and some of these other tabs are 4 spaces but looks like 2 spaces is the norm for the file.

Reply:
Good catch. Fixed